docs/resources/feature_state.md

```diff
@@ -1,19 +1,19 @@
 ---
 # generated by https://github.com/hashicorp/terraform-plugin-docs
-page_title: "flagsmith_flag Resource - terraform-provider-flagsmith"
+page_title: "flagsmith_feature_state Resource - terraform-provider-flagsmith"
 subcategory: ""
 description: |-
-  Flagsmith feature/ Remote config associated with an environment
+  Flagsmith Feature state/ Remote config value associated with an environment
 ---
 
-# flagsmith_flag (Resource)
+# flagsmith_feature_state (Resource)
 
-Flagsmith feature/ Remote config associated with an environment
+Flagsmith Feature state/ Remote config value associated with an environment
 
 ## Example Usage
 
 ```terraform
-resource "flagsmith_flag" "feature_1_dev" {
+resource "flagsmith_feature_state" "feature_1_dev" {
```

Comment:
Is this going to mean a breaking change?

Reply:
How do we handle that in terraform? We are currently basing this branch on main, should it be a release branch?